packages/CollabCodeEdit-Client.package/CCEModel.class/instance/buildGlobalStateAccountDisplayWith..st

```diff
@@ -0,0 +1,12 @@
+building
+buildGlobalStateAccountDisplayWith: aBuilder
+	^ aBuilder pluggableTextSpec new 
+		model: CCEAccount default;
```

Comment:
I would suggest to handle this as all our other singletons: create a instance variable in CCEModel and use this one for all usages of this singleton.
```suggestion
		model: self account;
```

Reply:
I know, in the existing code base at your state, this is not how we deal with singletons. in SessionManagment we do.